### DISCUSSION TITLE
Error occured when I, build CoRE4INET with inet 3.8.3 in OMNeT++ 6.0.2

### DISCUSSION THREAD

**Discussion by soreettihussein** [omnetpp/omnetpp#1130] [Q&A]
Hi All!
I received error message: cannot find inet framework in the usual location. you have to set the path to inet in inet_proj variable. However, I put both CoRE4INET and inet in the same location (workspace: sample). 
please can anyone help me!

**Comment by rhornig** [comment]
The inet folder still can have a different name. sometimes it contains a version number, sometimes not, depending on how you installed. You have to follow the installation instructions and set the variable as the message suggests.

or... just use `opp_env` to install the model.

**Reply by soreettihussein** [reply]
@rhornig  
thank you very much!! problem was solved.
